incubator/gitea/templates/configmap.yaml

```diff
@@ -0,0 +1,128 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  labels:
+    app: {{ template "fullname" . }}-config
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: {{ default "gitea" .Values.service.nameOverride }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "fullname" . }}-config
```

Comment:
All other resources are named `{{ template "fullname" . }}`. So should be the configmap.